sdk/python/feast/flags.py

```diff
@@ -0,0 +1,10 @@
+FLAG_ALPHA_FEATURES_NAME = "enable_alpha_features"
+FLAG_ON_DEMAND_TRANSFORM_NAME = "enable_on_demand_transforms"
+FLAG_PYTHON_FEATURE_SERVER_NAME = "enable_python_feature_server"
```

Comment:
Super nit: can we remove the `enable_` prefix? If i understand correctly the CLI usage is something like this:
```
feast alpha enable enable_on_demand_transforms
```

which seems verbose

Reply:
Done